corehq/apps/toggle_ui/migration_helpers.py

```diff
@@ -20,7 +22,11 @@ def move_toggles(from_toggle_id, to_toggle_id):
     for item in from_toggle.enabled_users:
         if item not in to_toggle.enabled_users:
             to_toggle.enabled_users.append(item)
-            update_toggle_cache(to_toggle_id, item, True)
+            namespace = None
+            if item.startswith(NAMESPACE_DOMAIN):
```

Comment:
not sure if worth it but these 3 lines could be extracted into a function like
```python
namespace, entry = parse_toggle(item)
```
which might be easier to understand and a little more DRY